packs/aws/README.md

```diff
@@ -1,32 +1,103 @@
-AWS Integration Pack
-=====
+# AWS Integration Pack
+
 The StackStorm AWS integration pack supplies action integration for EC2 and Route53.
 
-#### Setup
+## Configuration
 
 You will need to add a set of AWS credentials, and default zone to the config.yaml file:
 
-	---
-	setup:
-	  region: ""
-	  aws_access_key_id: ""
-	  aws_secret_access_key: ""
-	interval: 20
-	st2_user_data: ""
+```yaml
+---
+setup:
+  region: ""
+  aws_access_key_id: ""
+  aws_secret_access_key: ""
+interval: 20
+st2_user_data: ""
+ ```
 
 You can generate the access key and secret access key by following these directions:
 
 http://docs.aws.amazon.com/IAM/latest/UserGuide/ManagingCredentials.html#Using_CreateAccessKey
 
-##### st2_user_data
+* ``service_notifications_sensor.host`` - Listen host for the HTTP interface.
+* ``service_notifications_sensor.port`` - Listen path for the HTTP interface.
+* ``service_notifications_sensor.path`` - Path where the events need to be sent.
+
+## st2_user_data
 
 Optionally, you can set the user_data to set a default file to be used during new instance creation.  Put your user_data file somewhere accessible by the StackStorm user, and use the st2_user_data config option to set it.
 
-	st2_user_data: "/full/path/to/file"
+```yaml
+st2_user_data: "/full/path/to/file"
+ ```
 
 This file/script will be used for all invocations of the ec2_run_instances action
 
-#### Route53 Actions
+## Sensors
+
+### ServiceNotificationsSensor
+
+This sensor exposes a HTTP interface and listens for service event
+notifications which are delivered via AWS SNS service using HTTP(s) endpoint.
+
+Currently it supports event notifications generated by the S3 service -
+http://docs.aws.amazon.com/AmazonS3/latest/dev/NotificationHowTo.html
+
+Keep in mind that this sensor doesn't implement any kind of authentication.
```

Comment:
We currently don't do that so we should also add a similar note to the new relic sensor - /cc @manasdk